README.md

```diff
@@ -1,8 +1,63 @@
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL, NoSQL/MongoDB, Agile/Scrum, Git
+
+**Competent:** JavaScript, React, Nodejs, HTML/CSS, GitFlow, Docker
+
+**Familiar:** GCP, C/C++, Neo4j, Maven, AWS
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/linux_sql)]: Implented a clustering monitoring program which automates the collects hardware and cpu usage information from the linux servers using Bash scripts and Crontab. The data was stored in a postgresSQL database that was initialized using Dockers. It was later than queried to perform analytics on overall usage and identify downtimes during high server usage. We utilzed Git as a version control tool and GitFlow branching strategy.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/core_java)]:
+      
+  - Twitter App: In-progress
+  - JDBC App: Implemented JDBC to connect to a postgres database and execute CRUD operation following the Data Access Object (DAO) design pattern.
+  - Grep App: Created application to mimic linux grep command and utilized stream APIs to optimize efficiency. This was packaged using Maven and was Dockerized by building an image and uploading to docker hub.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo_OsamaShahid/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**Fifa Dashboard** [[GitHub](https://github.com/jarviscanada/jarvis_profile_builder)]: Created dashboard to find players team and matches played statistics that can potentially be used for sports betting. The frontend applciation was built using React and backend using Nodejs. The data was stored in MySQL database hosted on AWS-RDS.
```

Comment:
Double check grammar on these highlighted projects. You don't need to remove indefinite articles.